gradefetcher/gradefetcher.py

```diff
@@ -322,10 +321,11 @@ def grade_user(self, data, suffix=""):
         """
         Make a call to an external grader and retreive user's grade
         """
-        # Get EXTERNAL_GRADER from configuration
-        proxies = self.get_settings()["proxies"]
+
         # 1. If user in studio set authentication endpoint we call it
         try:
+            # Get EXTERNAL_GRADER from configuration
```

Comment:
@amirtds This is a huge try block with a very broad check for any Exception which would be good to handle a little bit better but ok I see this is an improvement